routes/update.js

```diff
@@ -0,0 +1,22 @@
+import express from 'express';
+import db from '../db/query.js';
+const router = express.Router();
+
+router.post('/', (req, res) => {
+    if (!req.body) {
+        res.render('error');
```

Comment:
What does `res.render('error')` result in? Does it send a 4xx, 5xx to the client?